build/distributions/CentOS/6/x86_64/DVD/ks/dvd/auto.cfg

```diff
@@ -34,6 +34,29 @@ else
 fi
 %end
 
+%pre-install
```

Comment:
Why do this and add logic to the simp-adapter rpm %post?  %post code seems sufficient.